docs/get-started/network-information.mdx

```diff
@@ -62,7 +62,8 @@ You can [set up your own node](/network/evm-nodes) if you need an endpoint witho
     </tr>
       <tr>
       <td>Snapshot and preimages site</td>
-      <td><a href="https://snapshots.eu.tzinit.org/etherlink-mainnet/">https://snapshots.eu.tzinit.org/etherlink-mainnet/</a></td>
+      <td><a href="https://snapshots.eu.tzinit.org/etherlink-mainnet/">Etherlink smart rollup snapshots</a></td>
+      <td><a href="https://snapshotter-sandbox.nomadic-labs.eu/etherlink-mainnet/">Etherlink EVM snapshots</a></td>
```

Comment:
It's a two column table so the formatting will not match.

```html
    <tr>
      <td>Etherlink smart rollup snapshots</td>
      <td><a href="https://snapshots.eu.tzinit.org/etherlink-mainnet/">https://snapshots.eu.tzinit.org/etherlink-mainnet/</a></td>
    <tr>
    </tr>
      <td>Etherlink EVM node snapshots</td>
      <td><a href="https://snapshotter-sandbox.nomadic-labs.eu/etherlink-mainnet/">https://snapshotter-sandbox.nomadic-labs.eu/etherlink-mainnet/</a></td>
    <tr>
```
and we can also link the preimages directly:

```html
    <tr>
      <td>Pre-images</td>
      <td><a href="https://snapshots.tzinit.org/etherlink-mainnet/wasm_2_0_0">https://snapshots.tzinit.org/etherlink-mainnet/wasm_2_0_0</a></td>
    </tr>
```

Idem for testnet.

Reply:
done, you are completely correct :slightly_frowning_face: